jkube-kit/common/pom.xml

```diff
@@ -135,6 +135,12 @@
       <groupId>com.networknt</groupId>
       <artifactId>json-schema-validator</artifactId>
     </dependency>
+      <dependency>
+          <groupId>io.fabric8</groupId>
+          <artifactId>kubernetes-server-mock</artifactId>
+          <version>4.10.2</version>
```

Comment:
I don't think you need to specify version here since it's covered by `<dependencyManagement>`

Reply:
anyway, If you are not going to mock Kubernetes API. You can remove this